source/frontend/src/features/mapSideBar/MotiInventoryContainer.test.tsx

```diff
@@ -76,7 +76,7 @@ describe('MotiInventoryContainer component', () => {
 
   it('shows the property information tab for inventory properties', async () => {
     mockAxios.onPost().reply(200, {});
-    mockAxios.onGet(new RegExp('/properties/*')).reply(200, { id: 9212434 });
+    mockAxios.onGet(new RegExp('/properties/concept/*')).reply(200, { id: 9212434 });
```

Comment:
if we are cleaning up the properties API we could also remove the '/concept' part for properties given we would not need that differentiation anymore